bundle/1.0.0-1/hive-operator.v1.0.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: OpenShift Optional
+    certified: 'false'
+    containerImage: quay.io/openshift-hive/hive:1.0.0
+    createdAt: '2020-03-23T22:36:00Z'
+    description: OpenShift cluster provisioning and management at scale.
+    support: Devan Goodwin
+  name: hive-operator.v1.0.0-1
```

Comment:
What's the -1 for here?

Reply:
mentioned this on slack. It's a release number -- the CSV release version, akin to an RPM release. I was thinking the packaging is independent of the payload. We might want to tweak something in the CSV only, and keep the binary the same, so 1.0.0-1 --> 1.0.0-2. Maybe not needed if we're checking in the CSV here. If we don't use a release number, then any small CSV tweak (like change the icon) requires a version bump for the hive image itself.

Reply:
This would imply editing directly in the community repo, and then overwriting later? I'm worried our dependency graph will break here if we start doing things outside what is committed. Is that possible? We need to consistently know what version we're replacing if I understand this right, in which case I think any CSV changes needed should be tagged and build as a new version of hive.

Reply:
> This would imply editing directly in the community repo, and then overwriting later

nope I was thinking that any small edits in /bundle wouldn't necessitate a version bump. New icon is a good example:
1. edit the CSV in this repo, set the icon
2. bump the release number, but not the hive version (1.0.0-1 --> 1.0.0-2)
3. commit CSV here
4. commit CSV to community-operators repo

This means no new tag after a minor CSV change, which I feel meh about. If you feel sfaer to bind operator and operand version (and icon change means 1.0.0 --> 1.0.1), I'm ok with it too.

Reply:
done